src/site/includes/image_detail_mobile_responsive.drupal.liquid

```diff
@@ -0,0 +1,20 @@
+{% assign alt = mediaImage.entity.image.alt %}
+<div class="vads-u-display--flex vads-u-flex-direction--column  medium-screen:vads-u-margin-bottom--2 vads-u-margin-bottom--4">
+    <dt class="vads-u-margin-bottom--2 medium-screen:vads-u-margin-bottom--0">
+        <{{image_title_tag}} class="{{image_title_classes }} medium-screen:vads-u-margin-top--0">{{ mediaImage.entity.entityLabel }}</{{image_title_tag}}>
```

Comment:
`image_title_tag` is declared here:
https://github.com/department-of-veterans-affairs/vets-website/pull/15270/files#diff-82e7af6315a8edf6ddfd71183d365b82c51fc78e99cde418334bad49c8191b04R46